src/utils/attack.ts

```diff
@@ -0,0 +1,60 @@
+import { Players } from "../store/players";
+import { Hit, Player, Room } from "../types/types";
+import { createAttackResponse, createChangeTurnResponse, createFinishResponse } from "./responses";
+import { LogMessages, Statuses } from "../constants/constants";
+import { updateWinners } from "../modules/update_winners";
+
+export const getRandomField = (player: Player) => {
+
+  while (true) {
+    const x = Math.floor(Math.random() * 10);
+    const y = Math.floor(Math.random() * 10);
+
+    if (!Players.isFieldHit(x, y, player)) {
```

Comment:
kudos for attention to details: in case of random attack only available cells are used.